app/controllers/sessions_controller.rb

```diff
@@ -5,7 +5,7 @@ def new
   end
 
   def create
-    user = User.find_by_email(params[:email])
+    user = User.find_by_email(params[:email].downcase)
```

Comment:
you could use a before_save callback in your model...i'd probably suggest that 👍 

```ruby
before_save :downcase_email

def downcase_email
  self.email = email.downcase
end
```